docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -718,13 +718,112 @@ To read and write a value of the Bluetooth descriptor:
       - write about requestAttMtuChange()
  -->
 ### Accessing the ATT MTU of connected device
+
+To get the ATT MTU value or request change of the ATT MTU value:
+
+1. Get BLE adapter:
+   ```
+   var adapter = tizen.bluetooth.getLEAdapter();
+   ```
+2. Start scannining and search for a device:
+   ```
+   var device;
+   adapter.startScan(function onsuccess(scannedDevice) {
+   if (device.name == "MyBLEDevice") {
+      device = scannedDevice;
+      console.log("Found device: " + device.name + ", address: " + device.address);
+   }});
+   ```
+3. When the device will be found, stop scanning:
+   ```
+   adapter.stopScan();
+   ```
+4. Connect to the device:
+   ```
+   function onerror(e) {
+      console.log("Failed to connect to device: " + e.message);
+   }
+
+   function onconnected() {
+      console.log("Connected to device");
+   }
+
+   device.connect(onconnected, onerror);
+   ```
+5. If the device has been connected call `getAttMtu()` function on the device object to get current ATT MTU value:
+   ```
+   var attMtu = device.getAttMtu();
+   ```
+6. If the device has been connected call `requestAttMtuChange()` function on the device object to request a change of the ATT MTU value. Pass desired ATT MTU value as an attribute:
+   ```
+   var newAttMtuValue = 64;
+   device.requestAttMtuChange(newAttMtuValue);
+   ```
+   > **Note**
+   > Call to `requestAttMtuChange()` function should be accepted only if both devices support new ATT MTU value.
```

Comment:
"should be accepted" - what does it mean, precisely?

Reply:
I rephrased this Note. to:
After calling `requestAttMtuChange()` function ATT MTU value change should be accepted if both devices support new ATT MTU value according to the BLE stanard.